src/css/colors.css

```diff
@@ -23,6 +24,8 @@
 	--color-purple-200: #e878ff;
 	--color-purple-300: #4830fd;
 
+	--color-pink-000: #fff1f8;
```

Comment:
I would name this `--color-pink-100`, to be consistent with the scale used for other colors.